src/store.rs

```diff
@@ -36,39 +36,55 @@ impl StoreParams for DefaultParams {
 pub trait Store: Clone + Send + Sync {
     /// Store parameters.
     type Params: StoreParams;
+    /// Temp pin.
```

Comment:
Doesn't need to be here, but I think it would be great to have some comment about what a Temp Pin is and what it is used for.